src/main.cpp

```diff
@@ -861,11 +861,13 @@ QString UsageArguments ( char **argv )
         "  -t, --notranslation   disable translation (use English language)\n"
         "  -v, --version         output version information and exit\n"
         "\nServer only:\n"
-        "      --mastermix       master mix server mode where all clients hear the\n"
-        "                        mix of the mix master (i.e. first conn. client after idle)\n"
-        "                        (note: the mix master can secretly solo others and\n"
-        "                        non-mix masters can still control their own as well as\n"
-        "                        the mix master's channel in their own mix)\n"
+        "      --mastermix       Master mix server mode where all clients hear the\n"
```

Comment:
I'd really rather _not_ have this length of description here.  It should be a _brief_ note about the option.  The proper place for documentation on a feature is the web site.

Reply:
Right now, we don't have a very good place for stuff that isn't in the GUI for servers. @ann0see put the QoS info in Tips & Tricks, so maybe best there for now. I've re-jigged the server docs to remove the compilation instructions (not yet merged), so maybe a separate section in there? But for now, I'd say put this mix stuff in Tips & Tricks.

Reply:
Yes, I think Tip & Tricks for now, much as it's a bit obscure.  Maybe add a direct link there to the server help menu?

Reply:
I totally agree, this is too much info for a CLI help message. Feel free to point me to better locations and I'm happy to both make the help message shorter as well as explain it more thoroughly elsewhere. :)

Reply:
You can put it in here and make a PR to the "changes" branch, I think: https://github.com/jamulussoftware/jamuluswebsite/blob/release/wiki/en/en-Tips-Tricks-More.md  This is what we did for the Quality of Service stuff.

Reply:
Done. [Here](https://github.com/jamulussoftware/jamuluswebsite/pull/425) we go. 
Will shorten the CLI help message now in this PR.